package.json

```diff
@@ -130,6 +131,7 @@
     "lodash": "^4.17.21",
     "npm-check-updates": "^16.1.0",
     "nx": "^14.8.3",
+    "nyc": "^15.1.0",
```

Comment:
This package doesn't seem well maintained. Are there any alternatives we can consider?

Reply:
https://github.com/istanbuljs/nyc
`nyc` is just the cli wrapper for istanbuljs. Istanbuljs is actively maintained (and already included in our project). The CLI has low activity on github, but the codebase itself has a very narrow scope and small footprint. The inactivity in `nyc` has more to do with IstanbulJS's project structure than maintenance status.

Reply:
Resolved this by using the underlying istanbul package directly in the coverage collection script.